### PR TITLE
Extract new strings (2022-10-13)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: en-US/firefox-ios.xliff [hunk 1]
```diff
@@ -3967,6 +3967,11 @@ Private Tab</target>
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="13.4.1" build-num="13F100"/>
     </header>
     <body>
+      <trans-unit id="ContextualHints.Toolbar.Bottom.Description.v107" xml:space="preserve">
+        <source>Move the toolbar to the top in settings if that’s more your style.</source>
+        <target>Move the toolbar to the top in settings if that’s more your style.</target>
```
Comment (Delphine) [inline en-US/firefox-ios.xliff:3972]: @electricRGB I think "settings" should be "Settings" here?
Side-note: Looking at the schedule, is this the only string needed for v107?
thanks!

Comment (electricRGB) [reply en-US/firefox-ios.xliff:3972]: Heya @Delphine. Sorry about getting back so late, I'm prepping a video for the All-Hands call and was only paying attention to that.
@nishant2718 can you take a look at this?

Nishant can also answer your question definitively - but I think the answer is yes. lol

Comment (electricRGB) [reply en-US/firefox-ios.xliff:3972]: Also, do we know what the procedure is for fixing this stuff now that we have the automated PR? Do I just close this PR and re-run the workflow?

Comment (nishant2718) [reply en-US/firefox-ios.xliff:3972]: I had a look at the designs again and "Settings" was removed from it 😱 . So there are two strings that we'll be pushing.

Comment (Delphine) [reply en-US/firefox-ios.xliff:3972]: Thanks both (and congrats on that talk Roux ;) )!
@nishant2718 Understood - please send update this when you can, and I will then expose strings on Pontoon that way :)
Oh and yes, you can close this PR, fix it in your repo, and then re-run the workflow

Comment (electricRGB) [reply en-US/firefox-ios.xliff:3972]: haha. tyty. Will close this and re-run the flow again when all is good on our side. Shold be done EoD. :)

**Comment by electricRGB** [comment]
Interesting. It force pushed to this. Is everything ok? I was just curious what would happen. lol

**Comment by Delphine** [comment]
> Interesting. It force pushed to this. Is everything ok? I was just curious what would happen. lol

Looks like it worked :) (thought about the fact it probably should, a bit after replying this morning).
So just to double check we are indeed removing the "Settings" string here, and adding two - correct?
Approving this in the meantime to expose the two strings :)

**Review by Delphine — APPROVED** [review]

**Comment by electricRGB** [comment]
> So just to double check we are indeed removing the "Settings" string here, and adding two - correct?
@nishant2718?

**Comment by nishant2718** [comment]
Yup - that word has been removed in final designs.

**Pull request merged**